cmake/compiler.cmake

```diff
@@ -116,89 +116,49 @@ set (CMAKE_CXX_FLAGS_RELWITHDEBINFO
 
 unset(CC_DEBUG_OPT)
 
-message(STATUS "Looking for libunwind.h")
-find_path(UNWIND_INCLUDE_DIR libunwind.h)
-message(STATUS "Looking for libunwind.h - ${UNWIND_INCLUDE_DIR}")
-
-if (UNWIND_INCLUDE_DIR)
-    include_directories(${UNWIND_INCLUDE_DIR})
-endif()
-
-set(CMAKE_REQUIRED_INCLUDES ${UNWIND_INCLUDE_DIR})
-check_include_file(libunwind.h HAVE_LIBUNWIND_H)
-set(CMAKE_REQUIRED_INCLUDES "")
-
-if(BUILD_STATIC AND NOT TARGET_OS_DARWIN)
-    set(UNWIND_LIB_NAME libunwind.a)
-else()
-    # libunwind can't be compiled on macOS.
-    # But there exists libunwind.dylib as a part of MacOSSDK
-    set(UNWIND_LIB_NAME unwind)
-endif()
-find_library(UNWIND_LIBRARY PATH_SUFFIXES system NAMES ${UNWIND_LIB_NAME})
-
 # Disabled backtraces support on FreeBSD by default, because of
 # gh-4278.
 # Disabled backtraces support on arm64 by default, because of
 # gh-6272, gh-6060 and gh-6222.
```

Comment:
Looks like this comment should be updated, shouldn't it?

Reply:
Nope, I will update it in scope of #6889 (which will close #6060).